meson.build

```diff
@@ -122,6 +124,7 @@ nntrainer_conf.set('PLUGIN_INSTALL_PREFIX', nntrainer_libdir / 'nntrainer')
 nntrainer_conf.set('INCLUDE_INSTALL_DIR', nntrainer_includedir / '..')
 nntrainer_conf.set('MEMORY_SWAP', nntrainer_enable_swap)
 nntrainer_conf.set('MEMORY_SWAP_PATH', nntrainer_swapdir)
+nntrainer_conf.set('MEMORY_SWAP_LOOKAHEAD', nntrainer_swap_lookahead)
```

Comment:
I'm not sure lookahead is set as in nntrainer configuration. How about the model parameter?

Reply:
OK. it removed.